client/src/components/Form/FormCard.vue

```diff
@@ -69,7 +62,20 @@ export default {
     data() {
         return {};
     },
+    computed: {
+        portletHeaderClasses() {
+            return {
+                "portlet-header": true,
+                "cursor-pointer": this.collapsible,
```

Comment:
Thanks for fixing this accessibility issue. It works much better now. Maybe we should add a scoped class here and set the brand/link text color and hover text-decoration of collapsible section titles to indicate that they can be clicked on.

Reply:
I actually dropped the title distinction intentionally since it's not specifically clickable (but the whole header is) to align more with how contentitems work -- do you think we need more than the icon and pointer to signify it expands?  Maybe just swapping the icon to an up/down chevron, instead of an eye?

Reply:
I think so, so basically still display the title styled as link, but this can be done in a follow-up when then the icon is switched to the chevron in dev. Thanks again for fixing this.

Reply:
Cool, I'll follow up with some options in dev.  Thanks!